src/angular/processflow/step-header.scss

```
@@ -0,0 +1,102 @@
+@import '../styles/common';
+
+.sbb-step-header {
+  overflow: hidden;
+  outline: none;
+  position: relative;
+  font-family: $fontSbbLight;
+  -webkit-tap-highlight-color: transparent;
+}
+
+.sbb-step-label {
+  display: inline-block;
+  @include ellipsis;
+  vertical-align: middle;
+  color: $sbbColorMetal;
+  cursor: default;
+
+  html:not(.sbb-lean) & {
+    font-size: pxToRem(20);
+    line-height: pxToRem(20);
+    padding-bottom: pxToRem(11);
+
+    @include mq($from: tabletPortrait) {
+      font-size: pxToRem(21);
+      line-height: pxToRem(21);
+      padding-bottom: pxToRem(14);
+    }
+    @include mq($from: desktop) {
+      font-size: pxToRem(21);
+      line-height: pxToRem(21);
+      padding-bottom: pxToRem(22);
+    }
+    @include mq($from: desktop4k) {
+      font-size: pxToRem(32);
+      line-height: pxToRem(34);
+      padding-bottom: pxToRem(32);
+    }
+    @include mq($from: desktop5k) {
+      font-size: pxToRem(42);
+      line-height: pxToRem(45);
+      padding-bottom: pxToRem(42);
+    }
+  }
+  html.sbb-lean & {
+    font-size: pxToRem(16);
+    line-height: pxToRem(16);
+    padding-bottom: pxToRem(5);
+  }
+}
+
+.sbb-step-text-label {
+  text-overflow: ellipsis;
+  overflow: hidden;
+}
+
+.sbb-step-label-active {
+  cursor: pointer;
+  color: $sbbColorBlack;
+
+  &::after {
+    content: '';
+    display: block;
+    position: absolute;
+    bottom: 0;
+    width: 0;
+    left: 50%;
+    height: pxToRem(1);
+    border-bottom: pxToRem(1) solid currentColor;
+    transition: width 0.3s, left 0.3s;
+
+    html:not(.sbb-lean) & {
+      @include mq($from: desktop4k) {
+        border-bottom-width: pxToRem(2);
+      }
+    }
+  }
+
+  .sbb-step-header:focus &,
+  .sbb-step-header:hover & {
+    color: $sbbColorRed125;
+
+    &::after {
+      left: 0;
+      width: 100%;
+    }
+  }
+}
+
+.sbb-step-label-selected {
+  &::after {
+    left: 0;
+    width: 100%;
+  }
+}
+
+.sbb-step-label-active.sbb-step-label-locked {
+  cursor: default;
+
+  &::after {
```

I forgot to undo the hover effect. The focus effect should persist.

```suggestion
  .sbb-step-header:hover & {
    color: $sbbColorBlack;
  }

  &::after {
```